api/textSnippet.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const router = express.Router();
+
+const txtgen = require('txtgen');
+router.post('/generateText', (req, res) => {
+
+    //gets the body of score from the fetch option.
+    const sum = req.body.score;
+
+    //generates a random text .
+    const textsnippet = txtgen.paragraph([sum])
+
+    // checks for letters and empty spaces, and bug preventing ..
+    if (sum === "") {
```

Comment:
You generate the text first and then validate the "sum" value? What if I am gonna send some weird value that breaks txtgen?

You need to use user specified values only after validation. Can't trust the users.